packages/flutter/lib/utils/passkeys.dart

```diff
@@ -23,6 +24,9 @@ class PasskeyUtils {
         ),
         _auth = Authenticator(true, true);
 
+  ///Used the getter to access the _opts in constructor for test purposes
+  PassKeysOptions get opts => _opts;
```

Comment:
May not be for test purposes only, since sensitive information is not part of the options

Reply:
ok